stub-idp/src/integration-test/java/uk/gov/ida/apprule/UserLogsInIntegrationTests.java

```diff
@@ -57,6 +58,16 @@ public void loginBehaviourTest() {
         authnRequestSteps.userConsentsReturnSamlResponse(cookies, false);
     }
 
+    @Test
+    @Ignore
+    public void testStaleSessionReaper() throws InterruptedException {
+        // set times in StaleSessionReaperConfiguration to 1s, run this test and check the log lines
+        for(int i=0;i<10;i++) {
+            authnRequestSteps.userPostsAuthnRequestToStubIdp();
+            Thread.sleep(1000);
+        }
+    }
```

Comment:
Is there a  way to do this without waiting 10 seconds? What if we just tested the module in isolation?

Actually, where are we checking the log lines?

Reply:
Maybe this test can go since I added a test for to JDBIIdpSessionRepositoryTest?